src/shared/styles/reset.css

```
@@ -0,0 +1,132 @@
+  ruby,
+  section,
+  summary,
+  time,
+  mark,
+  audio,
+  video {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    font-size: 100%;
+    font: inherit;
```

Biome (2.1.2)

[error] Unexpected shorthand property font after font-size



(lint/suspicious/noShorthandPropertyOverrides)